src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -29,21 +29,15 @@
   <ItemGroup>
     <_pastShimFiles Include="System.Globalization.Native.dylib" />
     <_pastShimFiles Include="System.Globalization.Native.so" />
-    <_pastShimFiles Include="System.IO.Compression.Native.a" />
```

Comment:
You need to leave the files in this ItemGroup, otherwise you'll cause conflict resolution to pick older versions of files from packages and either fail or act unpredictably.

This group can have files that don't exist on disk any more.

Reply:
Actually, you should probably add the removed file names to this list just to make sure that nothing gets broken by a change in the platform manifest.

Reply:
Ah, I see, I read the intention here completely backwards.